superset/dashboards/commands/create.py

```diff
@@ -38,7 +39,7 @@ def __init__(self, user: User, data: Dict):
         self._actor = user
         self._properties = data.copy()
 
-    def run(self):
+    def run(self) -> Optional[Model]:
```

Comment:
Isn't `BaseDAO.create` returning `Model`, ie. this should also do the same? And if so, the changes above should probably also reflect this.